src/test/Testcafe/package.json

```diff
@@ -11,6 +11,8 @@
   "license": "ISC",
   "devDependencies": {
     "minimist": "^1.2.0",
-    "testcafe": "^0.22.0"
+    "process": "^0.11.10",
+    "testcafe": "^0.22.0",
+    "tslint": "^5.11.0"
```

Comment:
I would recommend setting environment variables in "scripts". And use this package: https://www.npmjs.com/package/cross-env (So the setting of environment variables works on all OS's)

Reply:
Moved setting env variables to scripts :)